src/AutoRest.CSharp/Common/Decorator/DefaultDerivedSchema.cs

```diff
@@ -51,14 +51,6 @@ private static void BuildInternalDefaultDerivedType(ObjectSchema schema, Diction
             if (actualBaseSchema is null)
                 throw new InvalidOperationException($"Found a child poly {schema.Language.Default.Name} that we weren't able to determine its base poly from {string.Join(',', schema.Parents?.Immediate.Select(p => p.Name) ?? Array.Empty<string>())}");
 
-            //Since the unknown type is used for deserialization only we don't need to create if its an input only model
-            var hasXCsharpUsageOutput = !actualBaseSchema.Extensions?.Usage?.Contains("output", StringComparison.OrdinalIgnoreCase);
-            if (!actualBaseSchema.Usage.Contains(SchemaContext.Output) &&
-                !actualBaseSchema.Usage.Contains(SchemaContext.Exception) &&
-                (!hasXCsharpUsageOutput.HasValue ||
-                hasXCsharpUsageOutput.Value))
-                return;
```

Comment:
This is removed because since now we will generate model factory entries for the models without output usage, when this model is the base model with a discriminator and is abstract, we will need the unknown to construct the instance.
If this unknown is generated and never used anywhere (for instance, it does not meet the criteria of model factory), the post process will take care of it and remove it as well

Reply:
But now we have a bug on the post process that it is removed anyway.